exercises/tree-building/src/main/Record.java

```diff
@@ -0,0 +1,17 @@
+public class Record {
```

Comment:
Everything here (and in general) that's public could probably also be package private 🙂